index.js

```diff
@@ -40,8 +46,10 @@ const inputs = [
         validator: v => v.match(/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i)
     }),
     read({
-        prompt: 'Domain that the cert will be issued for, e.g. example.com:',
-        argument: 'domain'
+        prompt: 'Domain(s) that the cert will be issued for (separated by spaces), e.g. example.com www.example.com foo.example.com:',
+        argument: 'domain',
+        processValue: v => v === undefined || Array.isArray(v) ? v : v.trim().split(/\s+/),
+        validator: v => v !== undefined && v.length > 0 && v.every(item => typeof item === 'string' && item.length > 0)
```

Comment:
I'm not sure this is necessary either, given that `yargs` always returns strings (array of strings in this case), so there's no need to validate the type. Also, it shouldn't be necessary to validate presence (`item.length > 0`) here, the `read` function should do that (we are not validating presence here for any other argument).